app/models/bulkrax/csv_entry.rb

```diff
@@ -79,13 +79,17 @@ def build_export_metadata
       self.parsed_metadata['model'] = hyrax_record.has_model.first
       build_mapping_metadata
 
+      # TODO: fix the "send" parameter in the conditional below
+      # currently it returns: "NoMethodError - undefined method 'bulkrax_identifier' for #<Collection:0x00007fbe6a3b4248>"
       if mapping['collection']&.[]('join')
-        self.parsed_metadata['collection'] = hyrax_record.member_of_collections.map { |c| c.send(work_identifier)&.first }.compact.uniq.join(';')
+        self.parsed_metadata['collection'] = hyrax_record.member_of_collection_ids.join('; ')
+        #   self.parsed_metadata['collection'] = hyrax_record.member_of_collections.map { |c| c.send(work_identifier)&.first }.compact.uniq.join(';')
       else
         hyrax_record.member_of_collections.each_with_index do |collection, i|
-          self.parsed_metadata["collection_#{i + 1}"] = collection.send(work_identifier)&.first
+          self.parsed_metadata["collection_#{i + 1}"] = collection.id
+          #     self.parsed_metadata["collection_#{i + 1}"] = collection.send(work_identifier)&.first
         end
-      end        
+      end
```

Comment:
@sephirothkod and @orangewolf bringing your attention here. the code we merged into `main` earlier doesn't work. I'm using the singular "collection" version of the code that keeps with what braydon was working on, but I'm not using the "send" method because it gave me the error below.
this works for BL and since braydon is deploying OD from his branch, figured this is ok to go into `main` for the time being.
![Screen Shot 2021-09-09 at 11 05 36 PM](https://user-images.githubusercontent.com/29032869/132816536-0ec0d196-7d10-4c9b-a013-de16bd4dd6f9.png)